src/logic/pluginLibraryLogic.js

```diff
@@ -29,10 +26,18 @@ export const pluginLibraryLogic = kea({
         pluginLoading: [
             false,
             {
-                setPluginLoading: (_, { pluginLoading }) => pluginLoading,
+                [actions.openPlugin]: () => true,
+                [actions.setActivePlugin]: () => false,
```

Comment:
Why `actions.something` here and just `something` in the other reducers? You can omit the `actions.`